### DISCUSSION TITLE
Supported application servers

### DISCUSSION THREAD

**Discussion by maxjulius** [open-telemetry/opentelemetry-java-instrumentation#10044] [Q&A]
Hello, 
I am currently looking into the use of OpenTelemetry but I am quite a beginner. 
There is a page talking about the supported libraries, frameworks, application servers and JVM's.
For application servers it says: 'These are the application servers that the smoke tests are run against:'
In case my application server doesn't have a supported version, what does that mean? 
Does that mean it is not supported by OpenTelemetry and therefore I need to configure more manually? In case this is true, what would I need to configure manually? Or is it just that the tests run on that version, so these tests don't cover my version? In case this is true, what would be the effect for me then? What would I need to do?

Hopefully you can help me out, 
thank you and a good start to your week

**Comment by trask** [comment]
> Or is it just that the tests run on that version, so these tests don't cover my version?

yes

> In case this is true, what would be the effect for me then? What would I need to do?

try it out, and report an issue if it doesn't work for you
